### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by prasha23**
Added machine learning tutorial in python in Hindi

## What does this PR do?
Add resource(s) | Improve repo

### Why is this valuable (or not)?
Simple and step by step explanation.

### How do we know it's really free?
It is available on Youtube.
### For book lists, is it a book? For course lists, is it a course? etc.
Course.
## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -349,9 +349,11 @@
 * [Machine Learning](https://www.youtube.com/playlist?list=PLYwpaL_SFmcBhOEPwf5cFwqo5B-cP9G4P) - 5 Minutes Engineering
 * [Machine Learning Full Course](https://www.youtube.com/watch?v=IoZGSQ07e8g) - Bharani Akella, Great Learning (YouTube)
 * [Machine Learning in Hindi](https://www.youtube.com/playlist?list=PLPbgcxheSpE0aBsefANDYe2X_-tyJbBMr) - Codebasics Hindi
+* [Machine Learning Tutorial in Hindi 2022](https://www.youtube.com/playlist?list=PLfP3JxW-T70Hh7j17_NLzjZ8CejSPx40V) - Indian AI Production (YouTube)
```
Comment (shrn01) [inline courses/free-courses-hi.md:352]:
```suggestion
* [Machine Learning Tutorial using Python in Hindi 2022](https://www.youtube.com/playlist?list=PLfP3JxW-T70Hh7j17_NLzjZ8CejSPx40V) - Indian AI Production
```
Dont change the resource name

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -349,9 +349,11 @@
...
 * [Machine Learning Tutorials For Beginners Using Python in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ai6fAMHp-acBmJONT7Y4BSG) - CodeWithHarry (YouTube)
 
 
+
```
Comment (shrn01) [inline courses/free-courses-hi.md:356]:
```suggestion
```

2 lines are required between sections

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: courses/free-courses-hi.md (hunk 2 above)
Comment (prasha23) [reply courses/free-courses-hi.md]: All changes made as per your suggestion.

**Comment by prasha23** [comment]
@shrn01 Rectified linter error. Kindly merge the PR.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**